usagov/usagov-2021
# Usagov 993 taxonomy info in datalayer

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-993

## Description


## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- Login to the cms and do an import
- Do bin/drush cr in command line
- Refresh browser and go to any page and "View Page Source" you should see the datalayer information with breadcrumb text and url included
- Screenshot attached
<img width="1512" alt="Screenshot 2023-12-06 at 1 13 20 PM" src="https://github.com/usagov/usagov-2021/assets/5551880/5cd011d7-9ece-4b00-8aee-12efbdf233d1">

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
